docs/source/reference/tpu.rst

```diff
@@ -1,25 +1,30 @@
 .. _tpu:
 
+=========
 Cloud TPU
-================================
+=========
```

Comment:
How about we don't add "Overview" to encompass "Free TPUs via TPU Research Cloud (TRC)" and "Getting TPUs in one command"? The reason is these latter 2 headers are eye-catching, so I think they should appear by default in the TOC on the right hand side of the page.

Reply:
Actually I didn't notice there's a right hand side list. anyways I fixed this.